README.md

```diff
@@ -298,7 +299,7 @@ modules should be required in the subprocess collecting coverage:
 
 ## Configuring `nyc`
 
-Any configuration options that can be set via the command line can also be specified in the `nyc` stanza of your package.json, or within a `.nycrc` (or `.nycrc.json`) file:
+Any configuration options that can be set via the command line can also be specified in the `nyc` stanza of your package.json, or within a `.nycrc`, `.nycrc.json`, or `nyc.config.js` file:
 
 **package.json:**
```

Comment:
could remove this line since we are saying this can go in multiple places...

and come to think of it, it would be really nice if this was actually just a table with the option names, valid input types and values, and a definition... That's a lot of work though so I will happily merge this PR without that!